pkg/config/config.go

```diff
@@ -58,14 +59,14 @@ func (c Config) Blockchain() Blockchain {
 
 // Load attempts to load the config from the given
 // path for the given netMode.
-func Load(path string, netMode netmode.Magic) (Config, error) {
+func Load(path string, netMode netmode.Magic, relativePath string) (Config, error) {
```

Comment:
Let's make the argument variadic, then you don't need to change all occurances of `Load` and `LoadFile`. E.g.  `Load(path string, netMode netmode.Magic, relativePath ...string)` and `LoadFile(configPath string, relativePath ...string)`. Also, the documentation should be updated.